src/otx/algorithms/classification/adapters/mmcls/models/heads/custom_cls_head.py

```diff
@@ -89,6 +89,9 @@ def simple_test(self, img):
         if torch.onnx.is_in_onnx_export():
             return cls_score
         pred = F.softmax(cls_score, dim=1) if cls_score is not None else None
+        if pred.dtype == torch.bfloat16:
```

Comment:
Can we extract this pattern to a function, like cast_torch_fp32 or something like that?

Reply:
Yes, we can, then where is the location for the function `cast_torch_fp32` proper? Could you recommend?

Reply:
https://github.com/openvinotoolkit/training_extensions/pull/2574/commits/757a58b823eca1c3b193f72a1e76edc7bcf23e7b
I added the function `cast_bf16_to_fp32` in `src/otx/algorithms/common/utils/utils.py`.